x-pack/plugins/apm/common/constants.ts

```diff
@@ -41,3 +43,8 @@ export const ERROR_LOG_STACKTRACE = 'error.log.stacktrace';
 export const ERROR_EXC_MESSAGE = 'error.exception.message';
 export const ERROR_EXC_STACKTRACE = 'error.exception.stacktrace';
 export const ERROR_EXC_HANDLED = 'error.exception.handled';
+
+export const TRANSACTION_TYPE_MAP: StringMap<string> = {
+  request: 'Request',
+  'page-load': 'Page load'
+};
```

Comment:
Hm, I think if we have a `/common/constants` file it should probably be the place for any constant we want to define? If we want it to be an ES property file, we could rename and then create a more general constants file next to it. I'm not sure if i have buy in yet to remove this name forking overall, but if I do, then we can remove this altogether. If we need to keep it, I'll figure out where this can go.

Reply:
True, a rename to something specific might be a better idea. 
That being said, I don't think `TRANSACTION_TYPE_MAP` is actually a constant. It's a utility method for converting `transactionType` to a ui label, and is used as a dynamic lookup `TRANSACTION_TYPE_MAP[transactionType]` whereas `ERROR_EXC_HANDLED` is a constant used across the codebase.

Reply:
Btw to make matters worse we also have https://github.com/elastic/kibana/blob/948f4bc6bb306fc076faa25652a07044f6950582/x-pack/plugins/apm/public/constants/index.js 😱

Reply:
> Maybe having some more localized components/SomeComponent/constants.js could work for these kind of values

Yeah, I think that makes more sense for ui stuff like `TRANSACTION_TYPE_MAP` since it's closely tied to where it's used. We for instance have other methods that maps transaction types to other labels.
How are you handling values other than the ones in the map? Would it be an idea to use a switch statement with a default value?

Reply:
Cool, I removed them for now and will revisit if we need to re-implement this kind of relabeling after talking with the team tomorrow at the weekly.